apps/test/unit/templates/teacherNavigation/lessonMaterials/LessonMaterialsContainerTest.tsx

```diff
@@ -132,4 +134,157 @@ describe('LessonMaterialsContainer', () => {
         .length === 0
     );
   });
+
+  describe('resource links', () => {
+    let windowOpenMock: jest.SpyInstance<
+      Window | null,
+      Parameters<typeof utils.windowOpen>
+    >;
+
+    beforeEach(() => {
+      windowOpenMock = jest
+        .spyOn(utils, 'windowOpen')
+        .mockImplementation(() => null);
+    });
+
+    afterEach(() => {
+      windowOpenMock.mockRestore();
+      jest.resetAllMocks();
+    });
+
+    function viewResource(resourceName: string, resourceAction: string) {
+      const label = screen.getByText(resourceName);
+      const resourceRow = label.closest(
+        '[data-testid="resource-row"]'
+      ) as HTMLElement;
+      expect(resourceRow).toBeDefined();
+      const menuButton = resourceRow.querySelector('button') as HTMLElement;
```

Comment:
Nit: we could use [within()](https://testing-library.com/docs/dom-testing-library/api-within/) instead of querySelector here
```suggestion
      const menuButton = within(resourceRow).getByRole('button')
```